pkgs/development/python-modules/kornia-rs/default.nix

```
@@ -0,0 +1,63 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, rustPlatform
+, cmake
+, nasm
+}:
+
+buildPythonPackage rec {
+  pname = "kornia-rs";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "kornia";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-7toCMaHzFAzm6gThVLBxKLgQVgFJatdJseDlfdeS8RE=";
+  };
+
+  format = "pyproject";
```

```suggestion
  pyproject = true;
```
Please move this below `version`.